### DISCUSSION TITLE
Welcome to Red Discussions!

### DISCUSSION THREAD

**Discussion by greggirwin** [red/red#4747] [General]
## Welcome!
  We’re trying Discussions as a place to connect with our community, as we do Gitter today. It's an experiment.
  * Ask questions
  * Share ideas

 To get started, introduce yourself and join the conversation.

**Comment by rebolek** [comment]
Hi, I’m trying how this works.

**Reply by greggirwin** [reply]
I was able to delete Boris' test spam message, even though I liked it.
